src/platform/testing/e2e/auth.js

```diff
@@ -23,64 +23,68 @@ function setUserSession(token, client) {
 }
 
 /* eslint-disable camelcase */
-function initUserMock(token, level) {
-  mock(token, {
-    path: '/v0/user',
-    verb: 'get',
-    value: {
-      data: {
-        attributes: {
-          profile: {
-            sign_in: {
-              service_name: 'idme',
-            },
-            email: 'fake@fake.com',
-            loa: { current: level },
-            first_name: 'Jane',
-            middle_name: '',
-            last_name: 'Doe',
-            gender: 'F',
-            birth_date: '1985-01-01',
-            verified: level === 3,
+function getDefaultUserResponse(level) {
```

Comment:
Changes in here are to make the user mock customizable, so that we can pass different profile data if needed for e2e tests.